common/remote/rmtssh.cpp

```diff
@@ -340,13 +340,6 @@ class CFRunSSH: public CInterface, implements IFRunSSH
  
     void exec(unsigned i,unsigned treefrom)
     {
-        {
-            CriticalBlock block1(sect);
-            if (!dryrun) {
-                if (slaves.ordinality()>1)
-                    PROGLOG("%d: starting %s (%d of %d finished)",i,slaves.item(i),done.ordinality(),slaves.ordinality());
-            }
-        }
```

Comment:
This crit section shouldn't be necessary anymore since we won't be logging the start of each running thread.  If we want to insert this back into the output we can by appending this to the resultText buffer, but I don't think it's really necessary.